config/environments/development.rb

```diff
@@ -12,7 +12,7 @@
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = ENV.fetch('CACHE_CLASSES', 'true') == 'true'
+  config.cache_classes = ENV.fetch('CACHE_CLASSES', 'false') == 'true'
```

Comment:
I believe I had to change this to get a rails upgrade to play nice with cucumber or capybara. Have you run the CI locally to check?

Reply:
Ah yes, thanks!